dotcom-rendering/src/components/marketing/epics/Lines.tsx

```diff
@@ -0,0 +1,26 @@
+import { css } from '@emotion/react';
```

Comment:
Source has an SVG-based lines component - if our task is to also bring SDC components into better alignment with DCR/Source then this may be a good place to start? https://guardian.github.io/csnx/?path=/docs/source-react-components-development-kitchen_lines--docs

Reply:
Good point. There is a comment at the bottom of this file about not using the source component, but I don't know if it's still relevant. I'll investigate

Reply:
I've replaced it with source's `StraightLines`, which looks the same!